data/migrations/20211102125418_courses.js

```diff
@@ -40,6 +40,7 @@ exports.up = (knex) => {
       table.integer('number_of_sessions').notNullable();
       table.string('difficulty').notNullable();
       table.string('session_type').notNullable();
+      table.string('syllabus_link');
```

Comment:
In your video I thought you mentioned that the syllabus link was not nullable, but here it looks like it is, but maybe I am missing understanding the code?